notification-service/src/main/resources/application.yml

```diff
@@ -1,6 +1,8 @@
 spring:
   application:
     name: parodos-notification-service
+    title: Parodos Notification Service
+    version: 1.0.13-SNAPSHOT
```

Comment:
@pkliczewski pls note this is another location needs to be updated when releasing a version

Reply:
I think this [line](https://github.com/parodos-dev/parodos/blob/main/Makefile#L177) will handle it :)